### DISCUSSION TITLE
Kaboom change logo

### DISCUSSION THREAD

**Discussion by lajbel** [replit/kaboom#162] [General]
Now the style fits more with Kaboom, very cool

**Comment by lajbel** [comment]
Speaking of changes, I think we will all miss Mark, in the first season he meant a lot to everyone, but now bean is the main character

**Comment by slmjkdbtl** [comment]
Oh so you see what's happening on master :D Thanks for caring about mark 🙏 Mark will definitely have a place in the new version
